src/crypto/CHIPCryptoPAL.h

```diff
@@ -116,6 +117,30 @@ inline constexpr char kPIDPrefixForCNEncoding[]    = "Mpid:";
 inline constexpr size_t kVIDandPIDHexLength        = sizeof(uint16_t) * 2;
 inline constexpr size_t kMax_CommonNameAttr_Length = 64;
 
+inline constexpr uint8_t kFabricBindingVersionV1 = 1u;
```

Comment:
could this be an enum class? That way we do not pass in raw values.

Reply:
It also prevents passing in invalid values and we would not need to do some extra checks (except on callers if they read some header)